feat: added install fest docs

wongsitu opened the pull request:
Added new docs for install fest. If there's anything else we need to include let me know

wongsitu committed:
feat: added install fest docs

Anidion commented:
This is really great!
Only thing to add would be a WSL install guide, or `choco` commands for native Windows.
Might be worth having a section on how to use the terminal for brand new members as well, it can just be -
Search for Terminal with Spotlight and open it, then copy the command exactly and press enter to confirm it 
MacOS example since Linux users should know about the terminal lol

Niyousha99 commented on src/installation/README.md:
````diff
@@ -0,0 +1,185 @@
+Homebrew/homebrew-core (git revision 3b6; last commit 2021-07-26)
+```
+
+## Installing VSCode
````
Can we add a small list of generally recommended plugins for our projects like prettier and eslint?

wongsitu replied:
The goal of Installfest is to help new members with the initial setup. I think Eslint/Prettier guides should be added to the project guides, not here. It depends on a case-by-case basis 🤔. PMs using React won't need the same plugins as the ones using Vue.js, Node, or any other stack. Also, it's worth noting that some Project CLIs, like NextJS, come with their own plugins, and project members might want to stick with the default ones 🤷‍♂️.

Niyousha99 replied:
Good point, we do already recommend a couple plugins for PMs to consider in our Project Manager Guide. I created another issue to add it to our docs site too. https://github.com/techstartucalgary/Docs/issues/19

Thanks!

Niyousha99 commented on src/installation/README.md:
```diff
@@ -0,0 +1,185 @@
+
+make sure you team is using the same version of node for consistency and to avoid errors.
+
+## Installing Python
```
Let’s add Django as well since a couple PMs said they will be using it for their backend.

wongsitu replied:
✅ Done, added a Django initial setup guide

Niyousha99 replied:
Sweet thanks! Perhaps it would be better to merge it with the existing Django guide under the "Guides" section, to avoid overlap, I'll leave that up to your discretion!

wongsitu committed:
feat: added django installation guide

wongsitu committed:
feat: added wsl2 installation instructions

wongsitu committed:
feat: added linux basic ommands

wongsitu commented:
> This is really great! Only thing to add would be a WSL install guide, or `choco` commands for native Windows. Might be worth having a section on how to use the terminal for brand new members as well, it can just be - Search for Terminal with Spotlight and open it, then copy the command exactly and press enter to confirm it MacOS example since Linux users should know about the terminal lol

✅ Added WSL2 installation guide and some basic Linux commands/ learning resources.
I don't think Windows terminal commands are necessary, we should encourage them to use Linux as much as possible.

wongsitu committed:
Merge branch 'main' of https://github.com/techstartucalgary/Docs into feat/WaiKa/installfest-docs

wongsitu committed:
fix: commented django changes

wongsitu committed:
feat: merged django guide changes

wongsitu commented:
I updated the current Django guide with my changes. Some of the commands were outdated since homebrew installs Python 3 by default. Also added a section on how to connect Postgres with Django instead of using SQLite.
